web/Web/web/interfaces/CAL/functions.py

```diff
@@ -65,7 +65,7 @@ def check_docid_exists(session, doc_id):
         raise CALServerError(resp['status'])
 
 
-def add_session(session, seed_query, mode):
+def add_session(session, seed_query, mode, judgments):
```

Comment:
Let's rename `judgments` to `seed_ judgments` and have it have a default `judgments=[]`